linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/incorrect_stig_blank.fail.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "max-test-user:x:10020:10020:Test User for Password Expiration:/:/sbin/nologin" >> /etc/passwd
+echo "max-test-user:!!:18648:0::::" >> /etc/shadow
```

Comment:
I get `error` for this test scenario:
```
INFO - Script incorrect_stig_blank.fail.sh using profile (all) OK
ERROR - Rule evaluation resulted in error, instead of expected fixed during remediation stage 
```

Reply:
Should be fixed